rst2pdf/styles.py

```diff
@@ -6,6 +6,7 @@
 import os.path
 import sys
 import re
+import yaml
```

Comment:
This is a third-party import and should be grouped with the below.

```
stdlib

third-party

first-party
```

I don't think we've anything enforcing this here but that's what PEP8 suggests iirc.

Reply:
ah, thanks